pkg/option/config.go

```diff
@@ -1383,7 +1383,7 @@ type DaemonConfig struct {
 	XDPMode             string     // XDP mode, values: { xdpdrv | xdpgeneric | none }
 	HostV4Addr          net.IP     // Host v4 address of the snooping device
 	HostV6Addr          net.IP     // Host v6 address of the snooping device
-	EncryptInterface    string     // Set with name of network facing interface to encrypt
+	EncryptInterface    []string   // Set of network facing interface to encrypt over
```

Comment:
Is it worth renaming this variable to `EncryptInterfaces` to better reflect its new type?

Reply:
I think this re-raises the upgrade question which we earlier managed to convince ourselves was not too much of a concern. Given this is targeted as a fix for backport, I think we should try to limit such changes, and instead target such a change for a future minor (X.Y.0) release.

Reply:
ah the problem is if we do that we break backwards compatibility. Right now old configs will continue to work because a single string will be read as a len(EncryptInterface) == 1 slice. And new users can put a 'space' separated list in there and it will be consumed as a slice. If we rename it that breaks. I guess we could rename just the value in the struct and tell the CLI to use a different name and map them, but that seems even more confusing.

Reply:
Instead of renaming we could just introduce a new `EncryptInterfaces` flag and slowly deprecate `EncryptInterface`.